run/test/specs/disappearing_message_checks_android.spec.ts

```diff
@@ -0,0 +1,429 @@
+  );
+  // Check if the 'Tap to download media' config appears
+  // Click on config
+  await device2.clickOnElement("Untrusted attachment message");
+  await sleepFor(100);
+  // Click on 'download'
+  await device2.clickOnElement("Download media");
+  // Wait for 10 seconds
+  await sleepFor(10000);
+  // Check if GIF has been deleted on both devices
+  await device1.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Media message",
+    maxWait: 1000,
+  });
+  await device2.hasElementBeenDeletedNew({
+    strategy: "accessibility id",
+    selector: "Media message",
+    maxWait: 1000,
+  });
+  await closeApp(device1, device2);
+}
+
+async function disappearingLinkMessage(platform: SupportedPlatformsType) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const testLink = `https://nerdlegame.com/`;
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
+  await newContact(platform, device1, userA, device2, userB);
+  await setDisappearingMessage(device1, [
+    "1o1",
+    "Disappear after send option",
+    "10 seconds",
+  ]);
+  await device1.navigateBack(platform);
+  // Send a link
+  await device1.inputText("accessibility id", "Message input box", testLink);
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message sent status: Sent",
+    maxWait: 20000,
+  });
+  // Accept dialog for link preview
+  await device1.clickOnElement("Enable");
+  // No preview on first send
+  await device1.clickOnElement("Send message button");
+  // Send again for image
+  await device1.inputText("accessibility id", "Message input box", testLink);
+  await sleepFor(100);
+  await device1.clickOnElement("Send message button");
+  // Make sure image preview is available in device 2
+  await device2.waitForTextElementToBePresent({
+    strategy: "id",
+    selector: "network.loki.messenger:id/linkPreviewView",
+  });
+  // Wait for 10 seconds to disappear
+  await sleepFor(10000);
+  await device1.hasElementBeenDeletedNew({
+    strategy: "id",
+    selector: "network.loki.messenger:id/linkPreviewView",
+    maxWait: 1000,
+  });
+  await device2.hasElementBeenDeletedNew({
+    strategy: "id",
+    selector: "network.loki.messenger:id/linkPreviewView",
+    maxWait: 1000,
+  });
+  await closeApp(device1, device2);
+}
+
+async function disappearingCommunityInviteMessage(
+  platform: SupportedPlatformsType
+) {
+  const { device1, device2 } = await openAppTwoDevices(platform);
+  const communityLink = `https://chat.lokinet.dev/testing-all-the-things?public_key=1d7e7f92b1ed3643855c98ecac02fc7274033a3467653f047d6e433540c03f17`;
+  const communityName = "Testing All The Things!";
+  // Create user A and user B
+  const [userA, userB] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+  ]);
```

Comment:
maybe put at least the community into a  function that you call from each tests so if we have to change it, it's only in one place

getTestingCommunity() {

return {communityName, communityLink}
}

Reply:
You could also create a CONSTANTS object for communities that you import to make things prettier.